docs/source/backgroundechemtechniques.rst

```diff
@@ -0,0 +1,74 @@
+Electrochemical Techniques
+===========================
+
+This page is designed to assist users in deciding which technique would best suit a given experiment. The JUAMI
+potentiostat offers three techniques for testing electrochemical systems - linear sweep voltammetry, cyclic
+voltammetry, and chronoamperometry. An introduction to each technique is below.
+
+Linear Sweep Voltammetry
+-------------------------
+
+Linear sweep voltammetry applies a potential that sweeps from one potential to a final potential and measures the
```

Comment:
Maybe show abbreviation here (i.e. Linear Sweep Voltammetry (LSV)) since you use acronym later.